sg_otio/track_diff.py

```diff
@@ -510,7 +510,8 @@ def diffs_by_shots(self):
         """
         return self._diffs_by_shots
 
-    def old_clip_for_shot(self, for_clip, prev_clip_list, sg_shot, sg_version=None):
+    @staticmethod
```

Comment:
Why turning this into a `staticmethod`? Also, why not using a `classmethod` which means that you don't have to go everywhere in the code where this is called and remove `self`?